apps/meteor/client/lib/chats/data.ts

```diff
@@ -104,15 +115,15 @@ export const createDataAPI = ({ rid, tmid }: { rid: IRoom['_id']; tmid: IMessage
 		return true;
 	};
 
-	const findPreviousOwnMessage = async (message: IMessage): Promise<IMessage | undefined> => {
+	const findPreviousOwnMessage = async (message?: IMessage): Promise<IMessage | undefined> => {
 		const uid = Meteor.userId();
 
 		if (!uid) {
 			return undefined;
 		}
 
 		const msg = Messages.findOne(
-			{ rid, 'tmid': tmid ?? { $exists: false }, 'u._id': uid, '_hidden': { $ne: true }, 'ts': { $lt: message.ts } },
+			{ rid, 'tmid': tmid ?? { $exists: false }, 'u._id': uid, '_hidden': { $ne: true }, 'ts': { ...(message && { $lt: message.ts }) } },
```

Comment:
Is the method `findLastOwnMessage` still necessary after these changes?

Reply:
Removed unused functions.